app/src/oculusvr/cpp/DeviceDelegateOculusVR.cpp

```diff
@@ -539,6 +539,11 @@ DeviceDelegateOculusVR::EnterVR(const crow::BrowserEGLContext& aEGLContext) {
     vrapi_SetClockLevels(m.ovr, 4, 4);
     vrapi_SetPerfThread(m.ovr, VRAPI_PERF_THREAD_TYPE_MAIN, gettid());
     vrapi_SetPerfThread(m.ovr, VRAPI_PERF_THREAD_TYPE_RENDERER, gettid());
+    // Enable asynchronous time warp (ATW).
```

Comment:
nit: As I understand, ATW is always enabled. This set's a extra latency mode for ATW, doesn't enable ATW

Reply:
Thanks for correction.